closed/GensrcJ9JCL.gmk

```diff
@@ -45,6 +45,7 @@ $(J9JCL_SOURCES_DONEFILE) : $(AllJclSource)
 	@$(ECHO) Building OpenJ9 Java Preprocessor
 	@$(MKDIR) -p $(J9TOOLS_DIR)
 	$(MAKE) $(MAKE_ARGS) -C $(OPENJ9_TOPDIR)/sourcetools -f buildj9tools.mk \
+		JAVA_SPEC_VERSION=$(VERSION_FEATURE) \
```

Comment:
This isn't necessary if a phony target, say `jpp_jar` is defined in `buildj9tools.mk` which builds `jpp.jar`; the occurrence of `$(call FixPath,$(JPP_JAR))` below would be replaced by `jpp_jar`. The rules for that new target would be platform-independent.

Reply:
`jpp.jar` shares `BuildJar_template` function with a few other jar files `om.jar`, `uma.jar`, `j9vmcp.jar`, and `j9nls.jar`. A phony target like `jpp_jar` might have to duplicate some code from `BuildJar_template` which seem not desirable.

Just got another thought, removing `FixPath` from `$(call FixPath,$(JPP_JAR))` can match `$(JPP_JAR)` with the target within OpenJ9 `buildj9tools.mk`. This can avoid https://github.com/eclipse/openj9/pull/11543.
@keithc-ca does this sound a better approach?